src/network/network.cpp

```diff
@@ -619,7 +619,9 @@ bool Network::BeginClient(const char* host, unsigned short port)
 	server_address.Resolve(host, port);
 	status = NETWORK_STATUS_RESOLVING;
 
-	window_network_status_open("Resolving...");
+	char str_resolving[13];
```

Comment:
I don't know very much about OpenRCT2 code, but, what happens if the translated string is longer than 12 chars? Does format_string() handle this?